procedural/examples/foo-procedures/src/main/java/gov/nasa/ammos/aerie/procedural/examples/fooprocedures/procedures/StayWellFed.java

```diff
@@ -1,129 +0,0 @@
-package gov.nasa.ammos.aerie.procedural.examples.fooprocedures.procedures;
-
-import gov.nasa.ammos.aerie.procedural.scheduling.Goal;
-import gov.nasa.ammos.aerie.procedural.scheduling.annotations.SchedulingProcedure;
-import gov.nasa.ammos.aerie.procedural.scheduling.plan.EditablePlan;
-import gov.nasa.ammos.aerie.procedural.timeline.Interval;
-import gov.nasa.ammos.aerie.procedural.timeline.collections.profiles.Real;
-import gov.nasa.ammos.aerie.procedural.timeline.collections.profiles.Strings;
-import gov.nasa.ammos.aerie.procedural.timeline.payloads.LinearEquation;
-import gov.nasa.ammos.aerie.procedural.timeline.payloads.Segment;
-import gov.nasa.ammos.aerie.procedural.timeline.payloads.activities.DirectiveStart;
-import gov.nasa.jpl.aerie.merlin.protocol.types.Duration;
-import gov.nasa.jpl.aerie.merlin.protocol.types.SerializedValue;
-import org.jetbrains.annotations.NotNull;
-
-import java.util.List;
-import java.util.Map;
-
-/**
- * This goal creates regularly spaced Bite Banana activities (to stay well-fed).
- * This will cause the /fruit resource to drop far below zero, so it also creates
- * Grow Banana activities to fix that.
- *
- * More detailed algorithm:
- * 1. The goal is only applied when /producer == Dole. This is analogous to a mission phase.
- * 2. During the Dole phase, place a Bite Banana at most every `bitePeriodHours` apart.
- *    This step accounts for existing Bite Bananas and is essentially a cheap reimplementation
- *    of Recurrence Goal.
- * 3. Resimulate to see the effects this has on the /fruit resource
- * 4. We now want to prevent /fruit from dropping below 0. We could do this by placing
- *    a Grow Banana before the first time it becomes negative, and then resimulate, but
- *    that would take a lot of simulations. So instead, we place the activity and then
- *    mock the effect model of Grow Banana by adding a slanted step function to /fruit;
- *    then iterate.
- *
- *    Since we are doing this all in one goal, we know that we just created these Bites,
- *    and can safely anchor the new Grows to the Bites, if there is one. So at each
- *    `/fruit < 0` point, we query for a bite banana and anchor to it if possible.
- * @param bitePeriodHours
- */
-@SchedulingProcedure
-public record StayWellFed(double bitePeriodHours) implements Goal {
```

Comment:
I think this is actually a helpful example to keep around somewhere. Why did you delete all of them?

Reply:
Ah sorry, I had assumed `StayWellFed` made it into the e2e-test directory as well, I'll restore this examples project then 👍

Reply:
Oh, no I didn't use it in any tests because its a little to complicated to be a helpful unit test.